Sources/LeafKit/LeafSource/NIOLeafFiles.swift

```diff
@@ -5,6 +5,17 @@ import Foundation
 ///
 /// Default initializer will
 public struct NIOLeafFiles: LeafSource {
+    // MARK: - Public
+    
```

Comment:
* Annotations/Documentation/Rearrangment
* Adds `defaultExtension` parameter to initializer for `NIOLeafFiles`